tools/src/versioning/android/expoModulesTransforms.ts

```diff
@@ -0,0 +1,76 @@
+import escapeRegExp from 'lodash/escapeRegExp';
+
+import { FileTransforms } from '../../Transforms.types';
+import { packagesToKeep, packagesToRename } from './packagesConfig';
+import { deleteLinesBetweenTags } from './utils';
+
+function expoModulesBaseTransforms(abiVersion: string): FileTransforms {
+  return {
+    path: [
+      {
+        find: 'src/main/java',
+        replaceWith: `src/main/java/${abiVersion}`,
+      },
+      {
+        find: 'src/main/kotlin',
+        replaceWith: `src/main/java/${abiVersion}`,
```

Comment:
Is it intentional to replace `/kotlin/` with `/java/`❓ 
```suggestion
        find: 'src/main/kotlin',
        replaceWith: `src/main/java/${abiVersion}`,
```

Reply:
yes https://github.com/expo/expo/blob/main/tools/src/versioning/android/android-copy-expo-module.sh#L20